cicd/jenkins/kitchen-freebsd122-py3

```diff
@@ -0,0 +1,17 @@
+@Library('salt@master-1.9') _
+
+runTestSuite(
+    ami_image_id: 'ami-0c273f14bc9df57d1',
```

Comment:
This ami should come from promoting https://github.com/saltstack/salt-jenkins/pull/1682 after it has been merged.

Reply:
yeah, that's correct, so I created 2 PRs which should be merged in series